mne/decoding/search_light.py

```diff
@@ -44,8 +46,7 @@ def __init__(self, base_estimator, scoring=None, n_jobs=1):  # noqa: D102
         self.n_jobs = n_jobs
         self.scoring = scoring
 
-        if not isinstance(self.n_jobs, int):
-            raise ValueError('n_jobs must be int, got %s' % n_jobs)
+        _validate_type(self.n_jobs, Integral, 'n_jobs')
```

Comment:
Now we have to `import Integral` which is a bit annoying. It's probably safe enough to use `int` in your special-casing, or even `'int-like'` if we're worried there are cases we require an actual built-in `int` (though I doubt we have any of those currently).